### ISSUE TITLE
Consider add "text-align" option to core/post-excerpt block

### ISSUE THREAD

**Issue by diegosomar** [WordPress/gutenberg#44324]
## What problem does this address?
Post/Page excerpt is a non-format string field and we can't center the text.
The core/post-excerpt block has no option to center the text, like core/post-title block.

## What is your proposed solution?
Add a "text-align" button to core/post-excerpt block

![text-align-option](https://user-images.githubusercontent.com/11670836/191484628-104c3536-14d7-4dc4-9069-2bcfc0d2489b.png)

**Comment by carolinan** [comment]
Hi!
I can not reproduce this, I am able to align the text when I add the block.
Looking through the code, this was added two years ago.
Can you provide any more details?

**Comment by diegosomar** [comment]
Hello @carolinan, I recoreded a video to show.
Before recording, I tried to add the excerpt to a PAGE (post type) and it worked.

But when I went to create or edit a TEMPLATE (FSE), the option is no longer available. So the alignment option appears on the page, but not in the page template.

My env:

- WP: 6.0.2
- Gutenberg Plugin: 14.1.1

https://user-images.githubusercontent.com/11670836/191503899-c030bed0-bae1-4407-bf77-819d5d8a7ee0.mp4
